test/e2e/backups/sync_backups.go

```diff
@@ -54,88 +54,92 @@ func (b *SyncBackups) Init() {
 func BackupsSyncTest() {
 	test := new(SyncBackups)
 	client, err := NewTestClient()
-	if err != nil {
-		println(err.Error())
-	}
-	Expect(err).To(Succeed(), "Failed to instantiate cluster client for backup tests")
+	Expect(err).To(Succeed(), "Failed to instantiate cluster client for backup tests with error %v", err)
 
 	BeforeEach(func() {
 		flag.Parse()
 		if VeleroCfg.InstallVelero {
-			Expect(VeleroInstall(context.Background(), &VeleroCfg, "", false)).To(Succeed())
+			err = VeleroInstall(context.Background(), &VeleroCfg, "", false)
+			Expect(err).To(Succeed(), "Failed to install velero with error %v", err)
 		}
 	})
 
 	AfterEach(func() {
 		if VeleroCfg.InstallVelero {
-			Expect(VeleroUninstall(context.Background(), VeleroCfg.VeleroCLI, VeleroCfg.VeleroNamespace)).To(Succeed())
+			VeleroUninstall(context.Background(), VeleroCfg.VeleroCLI, VeleroCfg.VeleroNamespace)
```

Comment:
Seems should add "err = " at front of this line.

Reply:
OK， I've modified the code